dev/com.ibm.ws.ejbcontainer.core/resources/com/ibm/ejs/container/container.nlsprops

```diff
@@ -2223,10 +2223,10 @@ CNTR9425E_NO_URL_SPEC_SPECIFIED=CNTR9425E: The embeddable Enterprise JavaBeans (
 CNTR9425E_NO_URL_SPEC_SPECIFIED.explanation=The properties that the embeddable Enterprise JavaBeans (EJB) container uses contain a URL resource with no specification specified.
 CNTR9425E_NO_URL_SPEC_SPECIFIED.useraction=Add a specification property for this URL resource to the properties file.
 
-CNTR9426E_INCORRECT_ASYNC_ANNO=CNTR9426E: The {0} bean in the {1} module of the {2} application has one or more methods with the Jakarta Concurrent Asynchronous annotation.
-CNTR9426E_INCORRECT_ASYNC_ANNO.explanation=Jakarta Concurrent Asynchronous annotations are only valid for CDI beans, not EJBs.
-CNTR9426E_INCORRECT_ASYNC_ANNO.useraction=Remove the Jakarta Concurrent Asynchronous annotation from method.
+CNTR9426E_INCORRECT_ASYNC_ANNO=CNTR9426E: The {0} bean in the {1} module of the {2} application has one or more methods with the Jakarta Concurrency Asynchronous annotation, which must only be used on CDI beans and is not valid on Jakarta Enterprise Beans.
+CNTR9426E_INCORRECT_ASYNC_ANNO.explanation=Jakarta Concurrency Asynchronous annotations are only valid for CDI beans, not Jakarta Enterprise Beans.
+CNTR9426E_INCORRECT_ASYNC_ANNO.useraction=Remove the Jakarta Concurrency Asynchronous annotation from method.
```

Comment:
Sorry I didn't catch this trivial grammar improvement before - "from method" ought to be "from the method".  This also applies to CNTR9427W_INCORRECT_ASYNC_ANNO_INTERFACE.useraction